README.md

```diff
@@ -178,6 +178,15 @@ evaluation.run(model)
 
 > **Note:** for multilingual tasks, make sure your class also inherits from the `MultilingualTask` class like in [this](https://github.com/embeddings-benchmark/mteb-draft/blob/main/mteb/tasks/Classification/MTOPIntentClassification.py) example.
 
+## Leaderboard
+
+The MTEB Leaderboard is available [here](https://huggingface.co/spaces/mteb/leaderboard). To submit:
+1. Run your model on MTEB
+2. Format the json files into metadata using the script at `scripts/other/mteb_meta.py`. For example
+`python scripts/other/mteb_meta.py path_to_results_folder`, which will create a `mteb_metadata.md` file.
```

Comment:
We could move the file to `scripts/mteb_meta.py`, I think it'll be easier to use